aar_doc/templates/markdown.j2

```diff
@@ -22,14 +22,15 @@ Tags: {{ metadata.galaxy_info.galaxy_tags | join(', ') }}
 
 ### Entrypoint: {{ entrypoint }}
 
-{{ argument_specs[entrypoint].short_description }}
+{{ argument_specs[entrypoint].short_description | ansible_doc_markup }}
 
 {% if "description" in argument_specs[entrypoint] %}
 {%- if argument_specs[entrypoint].description is string -%}
-{{ argument_specs[entrypoint].description }}
+{{ argument_specs[entrypoint].description | ansible_doc_markup }}
 {% else %}
 {%- for line in argument_specs[entrypoint].description -%}
-{{ line }}
+{{ line | ansible_doc_markup }}
+
```

Comment:
Initially I was **very** opposed to this particular change. But then I read the spec again.
(see: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html#role-argument-validation)

> This can be a single string or a list of strings. In case this is a list of strings, every list element is a new paragraph.

So thank you for getting us to be actually compatible with the spec. :)

While checking this I also discovered that we currently only test for the string variant of this field. Can you please add tests for the list variant?

Reply:
There now also is a `multiline` test case.